src/function/aggregate/nested/histogram.cpp

```diff
@@ -0,0 +1,227 @@
+#include "duckdb/function/aggregate/nested_functions.hpp"
+#include "duckdb/planner/expression/bound_aggregate_expression.hpp"
+#include "duckdb/common/types/chunk_collection.hpp"
+#include "duckdb/common/pair.hpp"
+#include "duckdb/function/scalar/nested_functions.hpp"
+#include "duckdb/planner/expression/bound_function_expression.hpp"
+#include <map>
```

Comment:
If order is not required, unordered_map is preferred. Also use the common include: "duckdb/common/unordered_map.hpp"